test/test_manifest.json

```diff
@@ -12221,5 +12221,12 @@
     "rounds": 1,
     "disableFontFace": true,
     "type": "eq"
+  },
+  {
+    "id": "bug1978317",
+    "file": "pdfs/bug1978317.pdf",
+    "md5": "ae4f643ee9bb0fd725277a9d1e0fb1df",
+    "rounds": 1,
+    "type": "eq"
```

Comment:
Would a `type: load` perhaps suffice here given that before this patch the PDF file fails to load at all? The PDF file is quite large and takes some time to render, so if we're not interested in the rendering itself but only in the fact that the xref table can be parsed correctly then maybe a load test could save some time (but I'm not sure if it matters much in practice).